index.md

```diff
@@ -24,8 +24,18 @@ The system either uses the contents of the old system, or the new image; Never s
 Updates can be delayed or scheduled.
 This default behavior can be adapted or controlled by a larger management system.
 
-1. It should always be possible to factory reset back to either the known built behavior of the system or roll back to previous behavior if an updated image does not function correctly.
+1. If an update does not function correctly it is possible to roll back to the container image previously functioning before the update.
+
+1. State (including per-machine configuration) is preserved across updates.
```

Comment:
I am not sure about `/etc` but am sure @cgwalters can clarify.

Reply:
I like this change and the rest LGTM

Reply:
THanks!